app/hooks/features/index.ts

```diff
@@ -131,9 +132,12 @@ export function useAllFeatures(projectId, options: UseFeaturesOptionsProps = {})
       });
     })) : [];
 
+    // We want to return custom features first, but preserve the overall sorting
+    const partitionedParsedData = flatten(partition(parsedData, (feature) => feature.isCustom));
```

Comment:
Not really a fan of the variable name, but couldn't come up with a better name. Hopefully the comment clears any confusion.

Reply:
`sorted` / `sortedByCustomFeature` seems fine 🤔